system/include/webgpu/README.md

```diff
@@ -13,27 +13,28 @@ The C++ files are **currently** Dawn-specific, but included in Emscripten for
 better compatibility with Dawn: it has roughly the same API as Dawn's copy, but
 is included here because it is strongly tied to an exact `webgpu.h` revision.
 
-Dawn additionally autogenerates two "snippets" that are used in Emscripten:
-- `library_webgpu_enum_tables.js`, which is pasted into [`library_webgpu.js`](../../../src/library_webgpu.js)
-- `webgpu_struct_info.json`, which is pasted into [`struct_info.json`](../../../src/struct_info.json).
-
-The file `webgpu_cpp_chained_struct.h` is also needed from Dawn's source,
-though it changes infrequently.
-
-Once that's done, you need to update the auto-generated files with the commands below:
-
-```
-./tools/maint/gen_struct_info.py
-./tools/maint/gen_struct_info.py --wasm64
-./tools/maint/gen_sig_info.py
-```
+To update these bindings from Dawn:
+- Copy `webgpu_cpp_chained_struct.h` from Dawn's source
+- Build the `emscripten_bits_gen` target in Dawn (gn or CMake build)
+- Paste `library_webgpu_enum_tables.js` over the appropriate section of [`library_webgpu.js`](../../../src/library_webgpu.js)
```

Comment:
Could we link to the appropriate section to help?

Reply:
Not to the line number because it moves, but I can explain this better.
I am hoping to move this into another file so we can remove this manual step, but TBD.